1.1/rifgenerator.py

```diff
@@ -0,0 +1,112 @@
+import os
+import itertools
+
+# all
+relations_functions = [
+    # Simple Features Topological Relations
+    "sfEquals",
+    "sfDisjoint",
+    "sfIntersects",
+    "sfTouches",
+    "sfWithin",
+    "sfContains",
+    "sfOverlaps",
+    "sfCrosses",
+
+    # Egenhofer Topological Relations
+    "ehEquals",
+    "ehDisjoint",
+    "ehMeet",
+    "ehOverlap",
+    "ehCovers",
+    "ehCoveredBy",
+    "ehInside",
+    "ehContains",
+
+    # RCC8 Topological Relations
+    "rcc8eq",
+    "rcc8dc",
+    "rcc8ec",
+    "rcc8po",
+    "rcc8tppi",
+    "rcc8tpp",
+    "rcc8ntpp",
+    "rcc8ntppi",
+]
+
+geom_literals = [
+    "gmlLiteral",
```

Comment:
This should be the predicate instead of the datatype right? asGML, asWKT, etc.

Reply:
Thanks for finding and fixing!